spec/src/main/asciidoc/method-query.asciidoc

```diff
@@ -352,6 +399,18 @@ The return type of a Query by Method Name is determined as indicated in the foll
 | `find` accepting a `PageRequest` | `Page<E>` or `CursoredPage<E>` | For use with pagination
 |===
 
+[WARNING]
+====
+When working with NoSQL databases, the `count` operation may not be supported, depending on the database structure and capabilities:
+
+- **Key-Value Databases**: In general, the majority of Key-Value databases do not support counting. Their design focuses on retrieving values by keys rather than aggregating data.
+
+- **Other NoSQL Databases**: Support for the `count` operation varies significantly:
```

Comment:
This line shouldn't be at the same level as the following two bullet points. It's also unnecessary so I'll propose omitting it
```suggestion
```